jetstream/outcomes/firefox_desktop/incrementality_sponsored_tiles.toml

```diff
@@ -0,0 +1,18 @@
+friendly_name = "Sponsored Tiles"
```

Comment:
This should be a unique name (not strictly required, but will be confusing if it's not).